app/models/comment.rb

```diff
@@ -20,7 +20,7 @@ class Comment < ApplicationRecord
     maximum: ArchiveConfig.COMMENT_MAX,
     too_long: ts("must be less than %{count} characters long.", count: ArchiveConfig.COMMENT_MAX)
 
-  validate :check_for_spam
+  validate :check_for_spam, on: :create
```

Comment:
Should we include [AO3-6038](https://otwarchive.atlassian.net/browse/AO3-6038) in this release, since it's the same fix?

Reply:
I think it's only half a fix for AO3-6038 -- you can still reply to the comment, just not un-spam it in the process -- so it's better to leave it out. Maybe it would be better to update the issue.